src/modules/vtol_att_control/tiltrotor.cpp

```diff
@@ -100,6 +106,13 @@ Tiltrotor::parameters_update()
 	param_get(_params_handles_tiltrotor.elevons_mc_lock, &l);
 	_params_tiltrotor.elevons_mc_lock = l;
 
+	/* avoid parameters which will lead to zero division in the transition code */
```

Comment:
@LorenzMeier @SimonWilks What do you think of these changes? I thought it was better to make sure from the beginning that we don't have parameters which will lead to a zero division. If you check this with an if statement in the transition code then the code will not be executed but then you are also in a weird state.